build.gradle.kts

```diff
@@ -36,7 +36,7 @@ dependencies {
     
     // Client
     compile ( "no.nav.helse:dusseldorf-ktor-client:$dusseldorfKtorVersion")
-    compile ( "no.nav.helse:dusseldorf-oauth2-client:$dusseldorfKtorVersion")
+    compile ( "no.nav.helse:dusseldorf-oauth2-client:1.4.0.8634f4b")
```

Comment:
Hvis dette fikser feilen med java.lang.NoSuchMethodError: 'net.minidev.json.JSONObject com.nimbusds.jose.jwk.RSAKey.toJSONObject()'
bør den settes i dusselforf-ktor, og deretter oppdatere dusseldorf-ktor versjon her.

Reply:
Ser at no.nav.helse.dusseldorf-ktor-test-support må bruke en annen versjon, så slik jeg forstår det så må en av de få egen versjonering uansett.

Reply:
Meeen jeg er på litt tynn is her :P

Reply:
Hmm, hva tenker du her @fraadsbrandth ?

Reply:
Bør nok ikke ha en miks av versjoner, det kan nok skap noen rare scenarioer.

dusseldorf-ktor-test-support er renamet til dusseldorf-test-support ettersom ktor-spesifikke ting ble flyttet ut. Det er ganske lenge siden.

Så bør egentlig oppdatere til 1.4.0.8634f4b i prosjektet.

Avhengigheter må oppdateres regelmessig, litt uheldig her å gå fra 1.2.5.xxx. til 1.4.0.xxx (ettersom det kan inneholde mange endringer)